recipes/rubinenv/meta.yaml

```diff
@@ -76,7 +76,7 @@ requirements:
     # conda is required for runtime inspection of installed software
     - conda
     - cffi
-    - click <7.1
+    - click
```

Comment:
`>= 7.0` would better match the requirement. We know our code does not work with 6.x.